<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QWidget" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(144, 158, 184);</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
font: 13pt;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timefield">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>565</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
font: 13pt;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAddNew">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>640</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Añadir Cita</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnBack">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;
font-size: 32px;
background: #01BFFF;
</string>
   </property>
   <property name="text">
    <string>Preprarar cita</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblFeedback">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>741</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Color: green;
font:12pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboPekes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
background: white;
</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAddSpecialDate">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Añadior cita especial</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SpecialDateDesc">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
</string>
   </property>
   <property name="text">
    <string>Describe brevemente la cita</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
